service/hub/deploy/manifests/deploy.yaml

```diff
@@ -80,3 +77,55 @@ spec:
             - name: config-vol
               mountPath: /config
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: hub-service
+  name: hub-service-manager
+  namespace: hub-service-system
+spec:
+  ports:
+    - name: http
+      port: 5001
+      protocol: TCP
+      targetPort: 5001
+  selector:
+    app: hub-service
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: hub-service-ingress-tls
+  namespace: hub-service-system
+type: kubernetes.io/tls
+data:
+  tls.crt: | 
+    ...
+  tls.key: |
+    ...
+---
+apiVersion: networking.k8s.io/v1
```

Comment:
Split into single yaml to make it change to other gateway service easily(like change to apisix)

Reply:
ok